### ISSUE TITLE
TestCI GitHub Action fail when PR is not Kubefirst code

### ISSUE THREAD

**Issue by fharper** [kubefirst/kubefirst#903]
TestCI GitHub Action fail when PR is not Kubefirst code (i.e. documentations changes). It's an assumption, please validate it.

**Comment by fharper** [comment]
Will wait on the next failed to check the issue with @converge

**Comment by fharper** [comment]
So I got the issue https://github.com/kubefirst/kubefirst/actions/runs/3706168028/jobs/6281031747 with this PR https://github.com/kubefirst/kubefirst/pull/931 . Can you check @converge please?

**Comment by fharper** [comment]
Not an issue anymore, please reopen or create a new issue if it happens again.
